models/comments-model.js

```diff
@@ -47,8 +47,20 @@ function deleteCommentByIdModel(commentId) {
   });
 }
 
+function updateCommentById(req) {
+  const votes = req.body.inc_votes;
+  const commentId = req.params.comment_id;
+  return commentValidation(commentId)
+    .then(() => {
+      const sqlQuery = `UPDATE comments SET votes = $1 WHERE comment_id = $2 RETURNING *`;
+      return db.query(sqlQuery, [votes, commentId]);
```

Comment:
You're reassigning the votes rather than adding to them. Essentially you're doing votes = inc_votes, rather than votes += inc_votes :)